pyteal/ast/subroutine_test.py

```diff
@@ -97,9 +104,27 @@ def fnWithMixedAnns2(a: ScratchVar, b) -> Expr:
     def fnWithMixedAnns3(a: Expr, b: ScratchVar):
         return Return()
 
+    def fnWithMixedAnns4(a: ScratchVar, b, c: abi.Uint16) -> Expr:
+        return Return()
+
+    def fnWithAllABIAnns(
+        a: abi.Uint16,
+        b: abi.StaticArray[abi.Uint32, Literal[10]],
+        c: abi.DynamicArray[abi.Bool],
+        *,
+        output: abi.Bool,
+    ):
```

Comment:
As mentioned elsewhere, if at all possible, could we go with this format instead:
```python
    def fnWithAllABIAnns(
        a: abi.Uint16,
        b: abi.StaticArray[abi.Uint32, Literal[10]],
        c: abi.DynamicArray[abi.Bool],
    ) -> abi.Bool:
```

Reply:
(I meant with generics...)